cypress/features/regression/experimental/textbox.feature

```diff
@@ -0,0 +1,151 @@
+Feature: Experimental Textbox component
+  I want to change Experimental Textbox component properties
+
+  Background: Open Experimental Textbox component page
+    Given I open "Experimental Textbox" component page
+
+  @positive
+  Scenario Outline: Set placeholder to <placeholder>
+    When I set placeholder to "<placeholder>"
+    Then Textbox placeholder is set to "<placeholder>"
+      Examples:
+      | placeholder             |
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+      # @ignore because of FE-1447
+      # | <>                    |
+
+  @positive
+  Scenario: Check disabled checkbox for a Textbox component
+    When I check disabled checkbox
+    Then Textbox component is disabled
+
+  @positive
+  Scenario: Uncheck disabled checkbox for a Textbox component
+    When I check disabled checkbox
+      And I uncheck disabled checkbox
+    Then Textbox component is not disabled
+
+  @positive
+  Scenario: Enable readOnly checkbox for a Textbox component
+    When I check readOnly checkbox
+    Then Textbox component is readOnly
+
+  @positive
+  Scenario: Disable readOnly checkbox for a Textbox component
+    When I check readOnly checkbox
+      And I uncheck readOnly checkbox
+    Then Textbox component is not readOnly
+
+  @positive
+  Scenario Outline: Set fieldHelp to <fieldHelp>
+    When I set fieldHelp to "<fieldHelp>"
+    Then fieldHelp is set to "<fieldHelp>"
+      Examples:
+      | fieldHelp               |
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+  #     # @ignore because of FE-1447

```

Comment:
Why it's double commented ? Did you check does it work with that special character `<>`?

Reply:
fixed the double comment. Still doesn't work, related with different task